server/models/User.js

```diff
@@ -16,7 +16,7 @@ const userSchema = new mongoose.Schema({
     type: String,
     required: true
   },
-  register_date: {
+  registerDate: {
```

Comment:
Register darte could be the same as created at, which could be added to our NoSQL document by tagging on `{ timestamps: true }` as a second argument in the Schema constructor.

Ie,

```js
const userSchema = new mongoose.Schema({ . . . }, { timestamps: true })
```

Reply:
Thanks so much.